optimizely/client.py

```diff
@@ -12,10 +12,11 @@
 class Client(object):
   ALLOWED_REQUESTS = ['get', 'post', 'put', 'delete']
 
-  def __init__(self, api_key, api_base=BASE_URL):
+  def __init__(self, api_key, api_base=BASE_URL, token_type):
     # set API information
     self.api_key = api_key
     self.api_base = api_base
+    self.token_type = token_type 
```

Comment:
also, there should be validation that the provided type is one of the 2 supported types